todolist.html

```diff
@@ -32,5 +32,84 @@
                 <i class='icon icon--left fa fa-trash-o'></i> Delete All
             </button>
         </div>
+
+        <!-- Information Form for adding and editing tasks -->
```

Comment:
no real point to these comments, they don't help describe more complex snippets of code and will be replaced with code anyways. i'd remove them